Makefile

```diff
@@ -57,8 +52,7 @@ packages: rpm srpm
 
 # Python packaging
 wheel:
-	python setup.py bdist_wheel
-	python3 setup.py bdist_wheel
+	python3 -m build
```

Comment:
That's fair, though instead of installing setuptools (and twine for `upload`), it might be better to just go with `hatch build` and `hatch publish` instead.
Also, in the twine target, I believe you should always upload both sdist and wheel, not just wheel.

Reply:
Also I am not a fan of the makefile in general. The github CI would not use it anyway. This is mostly kept because it was requested.